react/features/base/video-transform/components/TransformAwareView.js

```diff
@@ -0,0 +1,286 @@
+// @flow
+
+import React, { Component } from 'react';
+import { View } from 'react-native';
+
+import { DEFAULT_TRANSFORM } from '../constants';
+import type { Transform } from '../constants';
+
+/**
+ * The max distance that the border of the transformed
+ * view can leave the screen border.
+ */
+const MAX_OFFSET = 100;
```

Comment:
Would the % here be more universal than pixels ?

Reply:
Not really, because 100 is a safe value to be able to avoid the filmstrip and the toolbox, but if you look closely it turns into percentage when zooming back (see the _calculateAndApplyTransformation function)